blog/2020-11-12-the-making-of-new-user-sdk-tutorials.md

```diff
@@ -2,7 +2,6 @@
 tags:
   - v1
   - temporal
-  - announcement
```

Comment:
removing this tag from this post so that the collection of posts with this tag is more consistent.